official-git/run-tests.sh

```diff
@@ -9,7 +9,7 @@
 set -e
 
 # This is the tag of the release that dgit originated
-TAG=v2.8.0
```

Comment:
If we are testing dgit against the official git 2.10 tests then can we state that we are compatible/aligned with that version?

Reply:
Sure, at some point we should see how high we can push it now that we have a reasonable number of tests running, 2.10 was just the minimum I had to push it in order to not have the test suite fail because it was insisting on having a grammar error

Reply:
(But "compatible" is a strong word to use...)